internal/http/routes/api/v1/auth_test.go

```diff
@@ -184,3 +185,51 @@ func TestRefreshHandler(t *testing.T) {
 		require.Equal(t, http.StatusAccepted, w.Code)
 	})
 }
+
+func TestSettingsHandler(t *testing.T) {
```

Comment:
I would add a test here that checks that the user configuration is correctly updated in database as well.